app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -23,6 +23,7 @@ import AcuantContext, { AcuantCaptureMode } from '../context/acuant';
 import AnalyticsContext from '../context/analytics';
 import DeviceContext from '../context/device';
 import FailedCaptureAttemptsContext from '../context/failed-capture-attempts';
+import FeatureFlagContext from '../context/feature-flag';
```

Comment:
I hate to do this to you - but I think you'll need to rebase / merge `main` and [use the new `selfieEnabled` context](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/document-capture/context/selfie-capture.tsx).

Reply:
✅ Yep, [did that](https://github.com/18F/identity-idp/pull/10156/commits/26f8af4c3bfacf933f6e5f49057c93d3b3996dfd). Thanks!